buildSrc/src/main/groovy/airbyte-python.gradle

```diff
@@ -80,6 +80,15 @@ class AirbytePythonPlugin implements Plugin<Project> {
             envPath = venvDirectoryName
             minPythonVersion '3.10'
 
+            // Amazon Linux support.
```

Comment:
📚 could use some elaboration on why we do this instead of just run python.

Reply:
Fair!